SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -224,8 +227,35 @@ void FSpatialGDKEditor::GenerateSnapshot(UWorld* World, FString SnapshotFilename
 	}
 }
 
+bool FSpatialGDKEditor::IsManualWorkerConnectionSet(const FString& LaunchConfigPath)
+{
+	FString FileContents;
+	FFileHelper::LoadFileToString(FileContents, LaunchConfigPath.GetCharArray().GetData());
+
+	const FRegexPattern ManualWorkerFlagPattern("\"manual_worker_connection_only\" *: *true");
+	FRegexMatcher ManualWorkerFlagMatcher(ManualWorkerFlagPattern, FileContents);
+
+	if (ManualWorkerFlagMatcher.FindNext())
+	{
+		UE_LOG(LogSpatialGDKEditor, Warning, TEXT("Launch configuration for cloud deployment has \"manual_worker_connection_only\" set to true. This means server workers will need to be connected manually."));
+		return true;
+	}
+
+	return false;
+}
+
 void FSpatialGDKEditor::LaunchCloudDeployment(FSimpleDelegate SuccessCallback, FSimpleDelegate FailureCallback)
 {
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	if (IsManualWorkerConnectionSet(SpatialGDKSettings->GetPrimaryLaunchConfigPath()))
```

Comment:
Can we not do this inside `IsDeploymentConfigurationValid` ?

Reply:
I think we shouldn't. It's perfectly valid to have this flag set.

Reply:
But we're checking if your deployment config is valid. Just for cloud deployments.

Reply:
Did it, hope you're happy.

Reply:
So happy